python/docs/source/user_guide/pandas_on_spark/supported_pandas_api.rst

```diff
@@ -1,1648 +0,0 @@
-..  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We'll have to add `python/docs/source/user_guide/pandas_on_spark/supported_pandas_api.rst` into `.gitgnore`.

Reply:
Sure! I just added it to gitignore. Also doctest CI function at the bottom of the code is added.